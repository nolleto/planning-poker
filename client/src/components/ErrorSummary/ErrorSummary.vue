<template>
  <div :class="[bootstrap.alert, bootstrap.alertDanger]" role="alert">
    <slot></slot>
  </div>
</template>

<script>
import styles from '@/styles'

export default {
  name: 'ErrorSummary',

  computed: {
    bootstrap: () => styles.bootstrap
  }
}
</script>
