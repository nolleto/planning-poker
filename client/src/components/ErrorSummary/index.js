export { default } from './ErrorSummary.vue'
