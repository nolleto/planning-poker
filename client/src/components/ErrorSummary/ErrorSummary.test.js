import ErrorSummary from './index'
import { shallowMount } from '@vue/test-utils'

const shallowComponent = () => {
  return shallowMount(ErrorSummary)
}

describe('ErrorSummary component', () => {
  let wrapper

  describe('Snapshots', () => {
    beforeAll(() => {
      wrapper = shallowComponent()
    })

    it('matches', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
