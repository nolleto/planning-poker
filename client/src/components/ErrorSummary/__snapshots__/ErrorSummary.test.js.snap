// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorSummary component Snapshots matches 1`] = `
<div
  class="alert alertDanger"
  role="alert"
/>
`;
